screen2.0/src/app/search/_ccredetails/ccredetails.tsx

```diff
@@ -48,7 +48,7 @@ export const CcreDetails: React.FC<CcreDetailsProps> = ({ accession, region, bio
         />
       )}
       {page === 3 && <Ortholog accession={accession} assembly={assembly} />}      
-      {page === 4 && <GeneExpression assembly={assembly} genes={geneList} />}
+      {page === 4 && geneList && <GeneExpression assembly={assembly} genes={geneList} />}
```

Comment:
Not loading gene expression if Linkedgenes data is not available as a workaround

Reply:
Thanks, this is a good workaround for now